package.json

```diff
@@ -25,7 +25,7 @@
     "new": "janus-cli new --do-not-edit-packages",
     "release": "multi-semantic-release",
     "prepare": "husky install",
-    "versions:bump": "backstage-cli versions:bump && find . -name 'package.json' ! -path '*/node_modules/*' -exec sed -i '' -e '/devDependencies/,/\\}/{ s/\"\\^/\"/; }' {} \\;"
+    "versions:bump": "backstage-cli versions:bump && find . -name 'package.json' ! -path '*/node_modules/*' -exec sed -i -e '/devDependencies/,/\\\\}/{ s/\\\"\\\\^/\\\"/; }' {} ; && yarn install"
```

Comment:
ah, I see you're removing the OSX sed implementation in favour of the one true gnu sed implementation. +1. 

you might be able to remove some of those extraneous `\\` slashes by using `sed -r` to enable extended regex